src/V3Options.cpp

```diff
@@ -1090,6 +1090,7 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc, char
     DECL_OPTION("-dumpi-", CbPartialMatchVal, [this](const char* optp, const char* valp) {
         m_dumpLevel[optp] = std::atoi(valp);
     });
+    DECL_OPTION("-dump-tree-dot", OnOff, &m_dumpTreeDot).undocumented();
```

Comment:
```suggestion
```

Reply:
In the `notify` function, we will need to set `m_dumpLevel["tree"]` if unset and `m_dumpLevel["tree-dot"]` is set (because `--dump-tree-dot` was passed), as the current `dumpTree()` functions (added very recently) created by `VL_DEFINE_DEBUG_FUNCTIONS` check only for "tree" (and the filename).

Reply:
Note this now needs to be handled as part of the partial match on "-dump-" anyway (as that is a prefix of this option), so the undocumented() will go away.